samples/04_gis_analysts_data_scientists/forecasting_air_temperature_in_california.ipynb

```diff
@@ -0,0 +1,2703 @@
+{
```

Comment:
<p>define precisely what each of the following parameters are using in the current model, rather than just describing the parameters, </p><p> </p><p><code>The primary input parameters required for the tool are: </code></p><ul><li><code>input_features&nbsp;: non spatial dataframe, feature layer, or spatial dataframe containing the primary dataset and the explanatory variables, if there are any</code></li><li><code>variable_predict&nbsp;: field name containing the y-variable to be forecasted from the input feature layer/dataframe</code></li><li><code>explanatory_variables&nbsp;: list of the field names as 2-sized tuples containing the explanatory variables as mentioned above. Since there are none in this example, it is not required here</code></li><li><code>index_field&nbsp;: field name containing the timestamp</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>